app/scripts/controllers/overview.js

```diff
@@ -188,7 +188,48 @@ angular.module('openshiftConsole')
       $scope.childServicesByParent[parentName].push(child);
     };
 
+    // Assign each service a score for sorting. Services with routes or app
+    // labels are considered more important.
+    var scoreService = function(service) {
+      var score = 0;
+      var name = _.get(service, 'metadata.name', '');
+      var routes = _.get($scope, ['routesByService', name], []);
+      var childServices = _.get($scope, ['childServicesByParent', name], []);
+
+      if (!_.isEmpty(routes)) {
+        score += 5;
+      }
+
+      if (_.has(service, 'metadata.labels.app')) {
+        score += 3;
+      }
+
+      if (!_.isEmpty(childServices)) {
+        score += 1;
+      }
+
+      if (ServicesService.isInfrastructure(service)) {
+        score -= 5;
```

Comment:
Sorry just remembered that wouldn't happen since those get tiled separately.  So maybe the question here is really should infrastructure services be getting rendered by a separate ng-repeat, just like we do with standalone services and unserviced stuff